output.proto

```diff
@@ -64,8 +64,13 @@ message SystemHealth {
       SENSOR_TILTED = 3;
       SENSOR_SUSPENDED = 4;
     }
+    message SensorInfo {
+      string topic = 1;
+      string sensor_name = 2;
+      SensorStatus status = 3;
```

Comment:
Could you move `status` to the top and set 1 to the id? It's subtle but I would like users to think the status is the main information in the message.

Reply:
yup